src/Form/AvatarChangeType.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Form;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Form\Extension\Core\Type\FileType;
+
+class AvatarChangeType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('avatar', FileType::class, [
+                'mapped' => false,
+                'label' => 'Pasirinkite naują nuotrauką',
+            ])
+        ;
+    }
+
+    public function configureOptions(OptionsResolver $resolver)
```

Comment:
this is same as no configureOptions method, you dont set any defaults. may as well remove